ios/FluentUI/People Picker/ContactView.swift

```diff
@@ -93,6 +98,7 @@ open class ContactView: UIView {
     private func setupLayout() {
         var constraints = [NSLayoutConstraint]()
         constraints.append(contentsOf: avatarLayoutConstraints())
+        translatesAutoresizingMaskIntoConstraints = false
```

Comment:
In general, only the superview or whoever is adding your view to a view hierarchy should be setting translatesAutoresizingMaskIntoConstraints. Do you need this line?

Reply:
Originally I placed this in `ContactCollectionViewCell.swift`, but you commented that it was probably not needed. I thought it would be better placed in `ContactView.swift` instead. I'll move it back to `ContactCollectionViewCell.swift`.

Yes, this line is needed. Without it, the `ContactView`'s don't show up properly.

Reply:
Originally, I was curious to know why it was needed since I didn't see where we were adding layout constraints to this view. This does need to go where the view is being added to another subview. Sorry for the miscommunication.

Reply:
All good. Thanks for the follow up 👍